app/src/main/graphql/project.graphql

```diff
@@ -32,3 +32,42 @@ query GetProjectAddOns($slug: String!, $locationId: ID!) {
     }
   }
 }
+
+query GetProjectComments($slug: String!, $cursor: String!){
+      project(slug: $slug) {
+        collaborators {
+          edges {
+            node {
+              id
+              name
+            }
+          }
+        }
+        comments(after: $cursor) {
+          edges {
+            cursor
+            node {
+              authorBadges
+              ...comment
```

Comment:
Great work with that fragment 👍

Reply:
small question here.
What do you think about moving `authorBadges` inside the comment fragment?. If I'm correct the badges are those tags like : "Superbaker" in the designs those tags will be diplayed in the comment card, so I guess it makes sense considering the badges as comment field